Dockerfile

```diff
@@ -1,53 +1,38 @@
-FROM ubuntu:22.04
+FROM debian:bookworm-slim
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV NODE_MAJOR=20
+ENV NPM_CONFIG_LEGACY_PEER_DEPS=true
+ENV PATH="/root/.cargo/bin:${PATH}"
```

Comment:
Necessary?

Reply:
I can drop the node & npm stuff, bt will keep adding cargo to the bath and the SNCOSMO env var, it can get confused without that

Reply:
I take that back we need all of that, I thought I was on the bun PR for some reason, we do need all these vars